tests/bootstrap.php

```diff
@@ -105,6 +105,7 @@
 include_once __DIR__ . '/GLPITestCase.php';
 include_once __DIR__ . '/DbTestCase.php';
 include_once __DIR__ . '/CsvTestCase.php';
+include_once __DIR__ . '/FormBuilder.php';
```

Comment:
This would not be required if the file was located inside `tests/src` and if the class was located inside the `Glpi\Tests` namespace.